mkJmod.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Das war für #18 gedacht. Ist jetzt eher obsolet. Ausgelagert in ein extra Projekt:
https://github.com/PIUGroup/informally-jmod-hub